.github/ISSUE_TEMPLATE/technical-debt.md

```diff
@@ -7,17 +7,31 @@ assignees: ''
 
 ---
 
-## Description 
-A clear and concise description of what the tech debt is and the reason of being created
+# Description 
 
-## Impact 
-Description of the current or possible impact of this tech debt.
+A clear and concise description of what the technical debt is, and the reason it should be improved.
 
-## Proposed solutions
-_How would you fix it?_
+# Impact 
+
+Description of the current or possible impact of this tech debt. Include both the positive and negative.
 
 ## Observations
 
-### Related files
+How would this change make things better?
+
+# Proposed solutions
+
+Do you have a solution, what would you recommend to fix it?
```

Comment:
This is awkwardly worded / punctuated. For whatever reason an alternative is not immediately springing to mind, but lets do better.